### PR TITLE
Recipes for pcli and pgspecial

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
License include prs
xref: https://github.com/dbcli/pgcli/pull/728
xref: https://github.com/dbcli/pgspecial/pull/38

**Comment by mariusvniekerk** [comment]
cc @synapticarbors @jakirkham Looks okay?

File: recipes/pgcli/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - pgcli.packages.pgliterals
+  commands:
+    - pgcli --help
+
+about:
+  home: http://pgcli.com
+  license: BSD-3-Clause
+  license_family: BSD
+  # xref: https://github.com/dbcli/pgcli/pull/728
+  #license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipes/pgcli/meta.yaml]: xref: https://github.com/dbcli/pgcli/pull/728

File: recipes/pgspecial/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  imports:
+    - pgspecial
+    - pgspecial.help
+
+about:
+  home: http://pgcli.com
+  license: BSD-3-Clause
+  license_family: BSD
+  # xref: https://github.com/dbcli/pgspecial/pull/38
+  # license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipes/pgspecial/meta.yaml]: xref: https://github.com/dbcli/pgspecial/pull/38

File: recipes/pgcli/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  entry_points:
+    - pgcli=pgcli.main:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pgspecial >=1.8.0
```
Comment (jakirkham) [inline recipes/pgcli/meta.yaml]: Normally we split these into separate PRs to avoid race conditions with the CIs. Though it is always possible to restart the builds.

**Comment by jakirkham** [comment]
Thanks @mariusvniekerk.
